geneticalg/geneticalg.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+
+import numpy as np
+
+
+def gen_chrom(max_seconds, video_list, video_lengths):
+    num_seconds = 100000000
+    while num_seconds > max_seconds:
+        chrom = np.random.choice([0, 1], size=(len(video_list),),
+                                 p=[4./5, 1./5])
+        num_seconds = sum(chrom*video_lengths)
+    return chrom
+
+
+def calc_duration(chrom, video_lengths):
```

Comment:
same here although kinda obvious that returns duration in I guess minutes (why bother converting to them?) ... but then actual manipulation is not entirely clear from the code since it depends on either chrom or video_lengths to be an `np.array`

Reply:
returned them in minutes for my own interpretation, but that's not necessary so I changed back to seconds.